src/lib/ursadb.py

```diff
@@ -15,11 +19,15 @@ def make_socket(self, recv_timeout=2000):
         socket.connect(self.backend)
         return socket
 
-    def query(self, query):
+    def query(self, query, taint):
```

Comment:
Since we're adding type hints in other functions of this class we might upgrade this one as well?

Reply:
Sure! I went ahead and added other missing type hints here